atom/browser/api/atom_api_web_contents.cc

```diff
@@ -671,6 +671,10 @@ void WebContents::DevToolsClosed() {
   Emit("devtools-closed");
 }
 
+void WebContents::DevToolsLoaded() {
+  Emit("-devtools-loaded");
```

Comment:
The `-` prefix is not needed here, we only add need it when there is an extra JavaScript wrapper.